Buildozer suggestions broken in rules_jvm_external?

Bazel 1.1.0 started warning about sha128 uses, which has pushed me to switch to `rules_jvm_external` from `rules_maven`.

Am I doing something wrong, or is Bazel now unable to give useful `buildozer` suggestions?

For example, in this case `A` and `B` both require Guava.

```python
java_library(
    name = "A",
    srcs = ["A.java"],
    deps = ["@maven//:com_google_guava_guava"],
)

java_library(
    name = "B",
    srcs = ["B.java"],
    deps = [
        ":A",
        "@maven//:com_google_guava_guava",
    ],
)

java_library(
    name = "B_no_dep",
    srcs = ["B.java"],
    deps = [
        ":A",
    ],
)
```

The error I get is not especially helpful at identifying the missing target that needs to be added to `B_no_dep`:

```
INFO: Writing tracer profile to '/private/var/tmp/_bazel_dan/55aaa51ea5620bd26651800f3bc9bddd/command.profile.gz'
INFO: Invocation ID: 57a6ccc4-bb94-4891-b8d4-6bdb1badfabd
INFO: Analyzed 3 targets (0 packages loaded, 0 targets configured).
INFO: Found 3 targets...
ERROR: /Users/dan/git-workspace/rules_jvm_demo/rules_jvm/BUILD:16:1: Building libB_no_dep.jar (1 source file) failed (Exit 1)
B.java:6: error: [strict] Using type com.google.common.collect.Lists from an indirect dependency (TOOL_INFO: "external/maven/v1/https/repo1.maven.org/maven2/com/google/guava/guava/26.0-jre/guava-26.0-jre.jar").
        System.err.println(Lists.reverse(A.INTEGERS));
                           ^
INFO: Elapsed time: 0.438s, Critical Path: 0.26s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
```

With `rules_maven`, I got a `buildozer` command that worked:

```
INFO: Analyzed 3 targets (1 packages loaded, 5 targets configured).
INFO: Found 3 targets...
ERROR: /Users/dan/git-workspace/rules_jvm_demo/rules_maven/BUILD:16:1: Building libB_no_dep.jar (1 source file) failed (Exit 1)
B.java:6: error: [strict] Using type com.google.common.collect.Lists from an indirect dependency (TOOL_INFO: "@com_google_guava_guava//jar"). See command below **
        System.err.println(Lists.reverse(A.INTEGERS));
                           ^
 ** Please add the following dependencies: 
  @com_google_guava_guava//jar to //:B_no_dep 
 ** You can use the following buildozer command: 
buildozer 'add deps @com_google_guava_guava//jar' //:B_no_dep 

INFO: Elapsed time: 1.469s, Critical Path: 1.19s
INFO: 3 processes: 1 darwin-sandbox, 2 worker.
FAILED: Build did NOT complete successfully
```

Am I doing something wrong in setting this up so that Bazel is not able to give me useful suggestions? Code: https://github.com/dhalperi/rules_jvm_demo

--- a/.bazelci/presubmit.yml
+++ b/.bazelci/presubmit.yml
@@ -10,14 +10,11 @@ tasks:
       - bazel run @unpinned_maven_install_in_custom_location//:pin
       - bazel run @duplicate_artifacts_test//:pin
       - tests/bazel_run_tests.sh
-    test_flags:
-      - "--//settings:stamp_manifest=True"
     test_targets:
       - "//..."
       # manual tests
       # TODO: Re-enable after fixing https://github.com/bazelbuild/rules_jvm_external/issues/375
       # - "//tests/com/jvm/external:UnsafeSharedCacheTest"
-      - "//tests/unit/manifest_stamp:diff_manifest_test" # https://github.com/bazelbuild/rules_jvm_external/issues/283
   rbe_ubuntu1604:
     shell_commands:
       - bazel run @unpinned_regression_testing//:pin
@@ -82,14 +79,11 @@ tasks:
       - bazel run @unpinned_maven_install_in_custom_location//:pin
       - bazel run @duplicate_artifacts_test//:pin
       - tests/bazel_run_tests.sh
-    test_flags:
-      - "--//settings:stamp_manifest=True"
     test_targets:
       - "//..."
       # manual tests
       # commented out until https://github.com/bazelbuild/rules_jvm_external/issues/610 is fixed for Java 11
       # - "//tests/com/jvm/external:UnsafeSharedCacheTest"
-      - "//tests/unit/manifest_stamp:diff_manifest_test" # https://github.com/bazelbuild/rules_jvm_external/issues/283
   windows:
     environment:
       # This tests custom cache locations.

--- a/settings/BUILD
+++ b/settings/BUILD
@@ -8,5 +8,5 @@ exports_files(["stamp_manifest.bzl"])
 
 stamp_manifest(
     name = "stamp_manifest",
-    build_setting_default = False,
+    build_setting_default = True,
 )

--- a/tests/unit/manifest_stamp/BUILD
+++ b/tests/unit/manifest_stamp/BUILD
@@ -30,9 +30,6 @@ diff_test(
         "//conditions:default": "guava_MANIFEST.MF.golden.unix",
     }),
     file2 = "guava_MANIFEST.MF",
-    # TODO(https://github.com/bazelbuild/rules_jvm_external/issues/283): Fix reproducibility issues.
-    # This test requires --//settings:stamp_manifest.
-    tags = ["manual"],
 )
 
 genrule(

